frontend/angular.json

```
@@ -28,7 +28,9 @@
               "node_modules/bootstrap/dist/css/bootstrap.min.css",
               "src/styles.css"
             ],
-            "scripts": [],
+            "scripts": [
+              "node_modules/bootstrap/dist/js/bootstrap.bundle.min.js"
+            ],
```

Se usar o carrossel do Bootstrap não precisa desse scripts pra passar as imagens.
```suggestion
            "scripts": [],
```